v3/src/components/graph/components/graph.tsx

```diff
@@ -228,12 +235,22 @@ export const Graph = observer(function Graph({graphController, graphRef, pixiPoi
           />
 
           {renderGraphAxes()}
-
-          <svg ref={plotAreaSVGRef}>
+          {/*
+            Note that this division into plotArea1 and plotArea2 SVG group elements, along with the separate handling of
+            the Pixi container, is due to a Safari-specific bug. Apparently, Safari renders the position of foreign
+            element content incorrectly if it or its parent is translated. The only workaround, as of January 2024, is
+            to use the X and Y attributes of the foreignElement tag itself. See:
+            https://www.pivotaltracker.com/story/show/186784214
+          */}
+          <g ref={plotArea1Ref}>
+            {/* Components rendered below the dots/points should be added to this group. */}
             {renderPlotComponent()}
-            <foreignObject ref={pixiContainerRef} x={0} y={0} width="100%" height="100%"/>
+          </g>
+          <foreignObject ref={pixiContainerRef} width="100%" height="100%"/>
```

Comment:
Not new to this PR, but I think the width and height are incorrect here -- this makes the pixi container the size of the entire component when it need only be the size of the plot area.

Reply:
I've just fixed it in the last commit.